environment.yml

```diff
@@ -33,3 +33,4 @@ dependencies:
   - matplotlib==3.0.0
   - ponyo==0.2
   - matplotlib-venn
+  - svgutils
```

Comment:
Been good to specify the version for svgutils and matplotlib-venn so you dont run into issues.